lib/system/supports-colors.js

```diff
@@ -37,10 +37,6 @@ module.exports = (function () {
     return true;
   }
 
-  if (process.stdout && !process.stdout.isTTY) {
```

Comment:
Moving this check below all the others means this will create messy text files when redirecting the output to a text file via `>output.txt`. If this happens will then depend on the platform. `win32` for example will have this problem, other platforms may not, depending on the order of the checks below.